theme/src/components/nav-drawer.js

```diff
@@ -118,23 +118,29 @@ function PrimerNavItems({items}) {
       >
         {item.children ? (
           <Details key={index}>
-            {({open, toggle}) => (
-              <>
-                <summary onClick={toggle} style={{cursor: 'pointer'}}>
```

Comment:
@joshblack - What do you think of this lint disable? I feel `summary` is not semantically right for this component. Should this ideally result in a navitems refactor to include lists and `treeitem`s? 
How much of a priority is primer docs over main github components when it comes to these refactors?